app/controllers/assets_controller.rb

```diff
@@ -9,7 +9,7 @@ class AssetsController < ApplicationController
   before_action :require_login, except: %i[index show latest radio listen_feed]
   before_action :check_new_user_abuse, only: %i[new create]
 
-  etag { "#{white_theme_enabled?}/#{current_user&.id}" }
+  etag { "#{current_user&.id}" }
```

Comment:
Doesn't even have to be a string at this point.

Reply:
Good call.